src/osemgrep/cli_ci/Meta.mli

```diff
@@ -0,0 +1,66 @@
+(** Collect information about the project from the environment. *)
+
+type 'e t = {
```

Comment:
I have no idea what meta.py is doing. Can you add a toplevel comment in the .mli explaining
the goal of this module. Is it use to collect project metadata to send metrics data to semgrep's server?
What is a project? Should we define a Project.ml? It is git related?

Reply:
Is it a project in the sense of the Semgrep app? Again maybe a Project.ml would be useful?

Reply:
> Is it a project in the sense of the Semgrep app? Again maybe a Project.ml would be useful?

As far as I understand, the main goal of `meta.py` is to aggregate metadata when `semgrep` is used by a CI (and, by this way, `semgrep` can lint a project for every PRs). We can surely rename it `Project` if you prefer. I don't have strong opinion about names.